drv/stm32xx-i2c-server/src/main.rs

```diff
@@ -476,3 +476,5 @@ fn configure_muxes(
         }
     }
 }
+
+include!(concat!(env!("OUT_DIR"), "/notifications.rs"));
```

Comment:
Are the notifications not used by name in this server? (Are there implicit assumptions about i2c notification numbers/masks somewhere else?)

Reply:
They're baked into the I2C configuration, e.g.
```rust
        [
            I2cController {
                controller: Controller::I2C2,
                peripheral: Peripheral::I2c2,
                notification: crate::notifications::I2C2_IRQ_MASK,
                registers: unsafe { &*device::I2C2::ptr() },
            },
        ]
```